nebula-logger/main/log-management/lwc/logEntryEventStream/logEntryEventStream.js

```diff
@@ -68,24 +75,9 @@ export default class LogEntryEventStream extends LightningElement {
         unsubscribe(this._subscription);
     }
 
-    handleLoggingLevelFilterChange(event) {
-        this.loggingLevelFilter = event.target.value;
-    }
-
-    handleOriginTypeFilterChange(event) {
-        this.originTypeFilter = event.target.value;
-    }
-
-    handleOriginLocationFilterChange(event) {
-        this.originLocationFilter = event.target.value;
-    }
-
-    handleLoggedByFilterChange(event) {
-        this.loggedByFilter = event.target.value;
-    }
-
-    handleMessageFilterChange(event) {
-        this.messageFilter = event.target.value;
+    handleFilterChange(event) {
```

Comment:
Awww yeah, I love this change - having only 1 common `change` function is much better 🚀